src/crystal/system/file.cr

```diff
@@ -47,6 +47,43 @@ module Crystal::System::File
     m | o
   end
 
+  LOWER_ALPHANUM = "0123456789abcdefghijklmnopqrstuvwxyz".to_slice
```

Comment:
Shouldn't this be private?

Reply:
It's in the `Crystal::System` namespace, so inherently non-public.